app/src/main/java/com/kickstarter/libs/KSCurrency.java

```diff
@@ -68,6 +68,25 @@ public KSCurrency(final @NonNull CurrentConfigType currentConfig) {
     return NumberUtils.format(currencyOptions.value(), numberOptions);
   }
 
+  /**
+   * Returns a currency string appropriate to the user's locale, chosenCurrency and project preferred currency.
+   *
+   * @param initialValue Value to display, local to the project's currency.
+   * @param project The project to use to look up currency information.
+   * @param roundingMode This determines whether we should round the values down or up.
+   */
+  public String formatWithRewardPreference(final float initialValue, final @NonNull Project project, final @NonNull RoundingMode roundingMode) {
```

Comment:
I'm not sure what a "reward preference" is. And we also use this method for the converted pledge and goal amounts. I think a better name could be "formatWithProjectCurrency" since I think that's what this method is doing.